src/lib/components/BeforeAfter.react.js

```diff
@@ -1,55 +1,137 @@
-import React, {Component} from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import BeforeAfterSlider from 'react-before-after-slider';
+import {ImgComparisonSlider} from '@img-comparison-slider/react';
 
 /**
- * A light wrapper of BeforeAfterSlider.
+ * Before After Image Slider based on https://github.com/sneas/img-comparison-slider
  */
-export default class BeforeAfter extends Component {
-
-    render() {
-        return (
-            <BeforeAfterSlider {...this.props}/>
-        );
-    }
-
-}
+const BeforeAfter = props => {
+    const {
+        id,
+        before,
+        after,
+        width,
+        height,
+        hover,
+        value,
+        direction,
+        keyboard,
+        beforeProps,
+        afterProps,
+        beforeClassName,
+        afterClassName,
+    } = props;
+
+    return (
+        <div id={id}>
+            <ImgComparisonSlider
+                hover={hover}
+                value={value}
+                direction={direction}
+                keyboard={keyboard}
+            >
+                <img
+                    slot="first"
+                    width={width}
+                    height={height}
+                    src={before}
+                    {...beforeProps}
+                    className={beforeClassName}
```

Comment:
Shouldn't we include the class in the "before/after" props? Or is there a reson why you left it out?

Reply:
I'm not sure what you mean.  This matches the props of the original component.

Reply:
I was just thinking if we should include the "className" in the "beforeProps" (and "afterProps")? :)

Reply:
OK, I get it now.  Your call.  I was trying to match the current props so fewer would be  deprecated, but including className in `beforeProps` and/or `afterProps` makes sense too.

Reply:
A minor thing - the static code analysis picked up an unused variable,

![image](https://user-images.githubusercontent.com/1623542/189391267-3a39ed31-4a36-46bb-b929-6370e1d08142.jpeg)

Reply:
oh darn, I thought I took that out.  I'll fix it

Reply:
Actually - I already did fix that - it's not in the current version of the file